openstack/variables.tfvars

```diff
@@ -0,0 +1,10 @@
+gcp_auth_file="<Path/to/your/gcp/service-account/credentials>"
```

Comment:
Is there a reason this one doesn't highlight the `gcp_auth_file` in blue? It looks like the variable declaration of all variables following are highlighted in blue.